### PR TITLE
Update release-notes.md

### PR DESCRIPTION
**Pull request by quantumedusa**

### CODE REVIEW

File: doc/release-notes.md [hunk 1]
```diff
@@ -1,3 +1,85 @@
+0.21.0+taproot0.1 Release Notes
+=========================
+
+Bitcoin Core version 0.21.0-based Taproot Client 0.1 is now available from:
```
Comment (luke-jr) [inline doc/release-notes.md]: Maybe drop "version" or move it before 0.1?

File: doc/release-notes.md [hunk 2]
```diff
@@ -1,3 +1,85 @@
+0.21.0+taproot0.1 Release Notes
+=========================
```
Comment (luke-jr) [inline doc/release-notes.md]: Extend to line length above?

File: doc/release-notes.md [hunk 3]
```diff
@@ -1,3 +1,85 @@
...
+
+- Taproot activation parameters: At blockheight 681408 (2021 Apr 29) a BIP 8 signaling period will begin with signal bit 2 and an activation threshold of 90% (1815/2016 blocks in a difficulty period) with LOT=true. The minimum activation height is set to block 709632 (2021 Nov 11),  with a timeout height of 760032 (2022 Oct 27). The latest activation height is block 762048 (2022 Nov 10). 
+
+Updated RPCs
+------------
+
+- getblockchaininfo has replaced all references in returned data to BIP9 with BIP8.
+
+0.21.0+taproot0.1 change log
+======================
```
Comment (luke-jr) [inline doc/release-notes.md]: Extend to line length above?

**Review by luke-jr — CHANGES_REQUESTED** [review]

File: doc/release-notes.md (hunk 1 above)
Comment (quantumedusa) [reply doc/release-notes.md]: ACK

**Review by quantumedusa — COMMENTED** [review]
ACK on all changes above.

**Review by luke-jr — APPROVED** [review]

**Comment by quantumedusa** [comment]
Good to go on notes

**Pull request merged**